### PR TITLE
[issue-38] Relicensing with Apache License 2.0

### LINKED ISSUE #38: Relicense jboss-common-beans with Apache License 2.0
Starting with 2.1.0.Final, jboss-common-beans is licensed with the https://www.apache.org/licenses/LICENSE-2.0 license.

### PR DESCRIPTION
**Pull request by yersan**
Fixes #38

### CODE REVIEW

**Comment by yersan** [comment]
CC @bstansberry

File: pom.xml [hunk 1]
```diff
@@ -1,11 +1,8 @@
...
-    2.1 of ~ the License, or (at your option) any later version. ~ ~ This software is distributed in the hope that it will be useful, ~ but WITHOUT 
-    ANY WARRANTY; without even the implied warranty of ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU ~ Lesser General Public 
-    License for more details. ~ ~ You should have received a copy of the GNU Lesser General Public ~ License along with this software; if not, write 
-    to the Free ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org. -->
+<!--
+    Copyright Red Hat
+    SPDX-License-Identifier: Apache-2.0
+-->
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
```
Comment (bstansberry) [inline pom.xml:8]: The license element in this file needs to be updated as well.

**Review by bstansberry — CHANGES_REQUESTED** [review]
Thanks @yersan -- just one more thing....

File: pom.xml (hunk 1 above)
Comment (yersan) [reply pom.xml:8]: Good catch @bstansberry , sorry I missed that one. It is fixed now.

**Review by bstansberry — APPROVED** [review]

**Comment by bstansberry** [comment]
Thanks @yersan

**Pull request merged**